gustaf/show.py

```diff
@@ -302,7 +302,9 @@ def _vedo_showable(obj, as_dict=False, **kwargs):
         cmap_kwargs["input_array"] = data_name
 
         # set cmap
-        vedo_obj.cmap(**cmap_kwargs)
+        # pass input_cmap as positional arg to support 2023.4.3.
+        # arg name changed in 2023.4.4
+        vedo_obj.cmap(cmap_kwargs.pop("input_cmap"), **cmap_kwargs)
```

Comment:
How stable is the development of vedo? It seems there are a lot of changes in the naming scheme regularly. 

I looked through the docs of vedo and found three instances of the function cmap() the name of the cmap was for all different. The Points object has `input_name`, Volume has `c`, and `Picture` has `name`.

Reply:
I see, I think we can stick with position based arguments for now